client/tekled.html

```diff
@@ -56,12 +56,12 @@ <h3 class="pageheader">Teknologiledelse - eksamen {{exam}}</h3>
   <div class="hero-unit well">
   {{#with currentQuestion}}
   <h3>Oppgave {{questionNum}} av {{numberOfQuestions}}</h3>
-  <p class="lead">{{{question}}}</p>
-  <button id="0" class="btn btn-default submit-answer"><p>a) {{{alternatives.[0]}}}</p></button><br />
-  <button id="1" class="btn btn-default submit-answer"><p>b) {{{alternatives.[1]}}}</p></button><br />
-  <button id="2" class="btn btn-default submit-answer"><p>c) {{{alternatives.[2]}}}</p></button><br />
-  <button id="3" class="btn btn-default submit-answer"><p>d) {{{alternatives.[3]}}}</p></button>
-  <p>{{#if explanation}} Forklaring: {{explanation}} {{/if}}</p>
+  <p class="lead">{{#mathjax}}{{{question}}}{{/mathjax}}</p>
```

Comment:
Setting the tag for the whole area had some side effects. The manual JQuery updates of the button states (red, green) didn't work. Was reset to default immediately.